student.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom)
```

Comment:
- [ ] Please pass the `age`, `name = "Unknown"`, `parent_permission = true` to the initialize method so you can pass it to the superclass. :smile: